src/runtime-rs/crates/hypervisor/src/device/device_manager.rs

```diff
@@ -472,6 +547,19 @@ pub async fn do_handle_device(
     Ok(device_info)
 }
 
+pub async fn do_update_device(
+    d: &RwLock<DeviceManager>,
+    update_config: &DeviceConfig,
```

Comment:
This should be `updated_config` here as well, as we want to avoid the use of verbs in the function params.

Reply:
@Apokleos Can you make this small fix? Otherwise the PR looks good to me.

Reply:
@amshinde Thx, sorry, I forget to correct this one.

Reply:
fixed